package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "wiki-plugin-graphviz",
-  "version": "0.2.2",
+  "version": "0.3.0-7",
+  "publishConfig": {
```

Comment:
Reminder that we'll want to change these details as we publish the final 3.0 version. But that will come after we've merged this PR.